README.md

```diff
@@ -204,7 +209,14 @@ Congratulations, if the last command worked, you now have kubernetes running in
    $ mv config.tmp ~/.kube/config
    $ kubectl config use-context k8s-lxc-admin@k8s-lxc
    ```
-   The second line does some magic to merge the admin access credentials into the existing `KUBECONFIG` file.
+   The second line does some magic to merge the admin access credentials into the existing `KUBECONFIG` file. If the last command did not work out make sure that your `~/.kube/config` contains context setup at least like below
```

Comment:
I don't think this improves the readme. If the context is there, why would if fail in the first place. And even if this check is positive, there are no instructions how to fix this or how to go on.

I'd remove this hunk.

Reply:
For me the problem was that the context was configured under the different name, and therefore setting kctl config use context to non-existing context didn't make that much sense. But I will remove that for now and we can reiterate on that